reproman/cmdline/main.py

```diff
@@ -132,42 +150,38 @@ def setup_parser(
         for _intfspec in _interfaces:
             # turn the interface spec into an instance
             lgr.log(5, "Importing module %s " % _intfspec[0])
-            _mod = import_module(_intfspec[0], package='reproman')
+            _mod = import_module(_intfspec[0], package="reproman")
             _intf = getattr(_mod, _intfspec[1])
             cmd_name = get_cmdline_command_name(_intfspec)
             # deal with optional parser args
-            if hasattr(_intf, 'parser_args'):
+            if hasattr(_intf, "parser_args"):
                 parser_args = _intf.parser_args
             else:
                 parser_args = dict(formatter_class=formatter_class)
-            # use class description, if no explicit description is available
-                parser_args['description'] = alter_interface_docs_for_cmdline(
-                    _intf.__doc__)
+                # use class description, if no explicit description is available
+                parser_args["description"] = alter_interface_docs_for_cmdline(_intf.__doc__)
             # create subparser, use module suffix as cmd name
             subparser = subparsers.add_parser(cmd_name, add_help=False, **parser_args)
             # all subparser can report the version
             helpers.parser_add_common_opt(
-                subparser, 'version',
-                version='reproman %s %s\n\n%s' % (cmd_name, reproman.__version__,
-                                                 _license_info()))
+                subparser, "version", version="reproman %s %s\n\n%s" % (cmd_name, reproman.__version__, _license_info())
```

Comment:
one more overly long line... we should set more sensible max width for black

Reply:
I prefer 100, but 120 was set in tox.ini for flake8 `max-line-length`